webpack/dev.config.js

```diff
@@ -1,10 +1,9 @@
-require('babel/polyfill');
+require('babel-polyfill');
 
 // Webpack config for development
 var fs = require('fs');
 var path = require('path');
 var webpack = require('webpack');
-var WebpackIsomorphicTools = require('webpack-isomorphic-tools');
```

Comment:
Well, [to my understanding](https://github.com/halt-hammerzeit/webpack-isomorphic-tools/tree/cb9edf3296b5d19e8024b38c0d00db87c2154fb4#mainjs) the non-plugin version of webpack-isomorphic-tools is only required for server-side patching of the `require` function, which should be covered [here](https://github.com/psalz/react-redux-universal-hot-example/blob/3c086e2093df8300ead4dd647468ecda650db4de/bin/server.js#L23).